apps/omg_watcher_info/lib/omg_watcher_info/pending_block_processor/storage.ex

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherInfo.PendingBlockProcessor.Storage do
+  @moduledoc """
+  Contains storage related functions of the PendingBlockProcessor
+  """
+
+  alias OMG.WatcherInfo.DB.Block
+  alias OMG.WatcherInfo.DB.PendingBlock
+
+  def get_next_pending_block() do
```

Comment:
specs would be great. easier to read. (I know you added specs in PendingBlock already).